onnxmltools/convert/keras/operator_converters/AdvancedActivation.py

```diff
@@ -11,20 +11,21 @@
 
 def convert_keras_advanced_activation(scope, operator, container):
     op = operator.raw_operator
+    keras_name = scope.get_unique_operator_name(operator.raw_operator.name)
     if isinstance(op, activations.LeakyReLU):
         alpha = op.get_config()['alpha']
         apply_leaky_relu(scope, operator.input_full_names[0], operator.output_full_names[0], container,
-                         operator_name=operator.full_name, alpha=alpha)
+                         operator_name=keras_name, alpha=alpha)
```

Comment:
I didn't find the place where operator_name was referenced. Did I miss anything?

Reply:
operator_name is a named parameter to the apply_leaky_relu function. All apply_* functions have this parameter that allows the caller to set the name of the node container that is created.